prow/cmd/plank/README.md

```diff
@@ -21,7 +21,9 @@ plank:
   job_url_template: 'https://<domain>/view/<bucket-name>/pr-logs/pull/{{.Spec.Refs.Repo}}/{{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}}/{{.Spec.Job}}/{{.Status.BuildID}}'
   report_template: '[Full PR test history](https://<domain>/pr-history?org={{.Spec.Refs.Org}}&repo={{.Spec.Refs.Repo}}&pr={{with index .Spec.Refs.Pulls 0}}{{.Number}}{{end}})'
   default_decoration_configs:
-    '*':
+  # All entries that match a job are used, later entries override previous values.
```

Comment:
Yeah, the alternative would be for both formats to be included under different names which wouldn't be ideal either. To be honest the generated config doc doesn't seem to provide any value over what the godocs provide since all the field values are empty. Godocs + real world examples or samples are what I would reference today.

Reply:
The generated config contains the very same docs the godoc fields have while using the json field names, so IMHO it is a lot more useful than godocs.